tests/pytorch/iris.py

```diff
@@ -58,5 +60,24 @@ def test_step(self, batch, batch_idx):
         self.log("test_acc", self.test_acc.compute())
 
 
+class IrisClassificationWithoutValidation(IrisClassificationBase):
```

Comment:
Defined a new lightning-module class that doesn't override `validation_step` for testing